src/main/java/com/paneedah/mwc/CustomSpawnEggs.java

```diff
@@ -7,33 +7,33 @@
 
 public class CustomSpawnEggs {
 	
-	public static HighIQSpawnEgg TURRET_EGG;
-	public static HighIQSpawnEgg TURRETUPGRADED_EGG;
-	public static HighIQSpawnEgg TURRETSILENCED_EGG;
-	
-	public static void init(Object mod, CompatibleFmlPreInitializationEvent event, CommonProxy proxy) {
-		TURRET_EGG = new HighIQSpawnEgg.Builder()
-				.withID(1)
-				.withItemName("turret")
-				.withEntitySpawnName("turret")
-				.withBlockPredicate(b -> b.getRegistryName().toString().equals(ModReference.id + ":turret_base"))
-				.withCreativeTab(ModernWarfareMod.AssaultRiflesTab)
-				.build();
-		
-		TURRETUPGRADED_EGG = new HighIQSpawnEgg.Builder()
-				.withID(2)
-				.withItemName("turretupgraded")
-				.withEntitySpawnName("turretupgraded")
-				.withBlockPredicate(b -> b.getRegistryName().toString().equals(ModReference.id + ":turret_base"))
-				.withCreativeTab(ModernWarfareMod.AssaultRiflesTab)
-				.build();
-		
-		TURRETSILENCED_EGG = new HighIQSpawnEgg.Builder()
-				.withID(3)
-				.withItemName("turretsilenced")
-				.withEntitySpawnName("turretsilenced")
-				.withBlockPredicate(b -> b.getRegistryName().toString().equals(ModReference.id + ":turret_base"))
-				.withCreativeTab(ModernWarfareMod.AssaultRiflesTab)
-				.build();
-	}
+//	public static HighIQSpawnEgg TURRET_EGG;
+//	public static HighIQSpawnEgg TURRETUPGRADED_EGG;
+//	public static HighIQSpawnEgg TURRETSILENCED_EGG;
+//
+//	public static void init(Object mod, CompatibleFmlPreInitializationEvent event, CommonProxy proxy) {
+//		TURRET_EGG = new HighIQSpawnEgg.Builder()
+//				.withID(1)
+//				.withItemName("turret")
+//				.withEntitySpawnName("turret")
+//				.withBlockPredicate(b -> b.getRegistryName().toString().equals(ModReference.id + ":turret_base"))
+//				.withCreativeTab(ModernWarfareMod.AssaultRiflesTab)
+//				.build();
+//
+//		TURRETUPGRADED_EGG = new HighIQSpawnEgg.Builder()
+//				.withID(2)
+//				.withItemName("turretupgraded")
+//				.withEntitySpawnName("turretupgraded")
+//				.withBlockPredicate(b -> b.getRegistryName().toString().equals(ModReference.id + ":turret_base"))
+//				.withCreativeTab(ModernWarfareMod.AssaultRiflesTab)
+//				.build();
+//
+//		TURRETSILENCED_EGG = new HighIQSpawnEgg.Builder()
+//				.withID(3)
+//				.withItemName("turretsilenced")
+//				.withEntitySpawnName("turretsilenced")
+//				.withBlockPredicate(b -> b.getRegistryName().toString().equals(ModReference.id + ":turret_base"))
+//				.withCreativeTab(ModernWarfareMod.AssaultRiflesTab)
+//				.build();
+//	}
```

Comment:
Not related to props and anyway should not be removed as I said in #54

Reply:
turrets are being removed apparently with the props paneedah said

Reply:
> > turrets are being removed apparently with the props paneedah said
> 
> No, the sister mod for props will only contain nonfunctional blocks that's the definition of a prop

In a different mod